modules/core/src/main/java/org/apache/ignite/internal/replicator/ZonePartitionId.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.replicator;
+
+import java.util.Objects;
+
+/**
+ * The class is used to identify a zone replication group id for a given partition.
+ */
+public class ZonePartitionId implements ReplicationGroupId {
+
+    /** Zone id. */
+    private final int zoneId;
+
+    /** Partition id. */
+    private final int partId;
+
+    private final int tableId;
```

Comment:
Please either add javadoc to the tableId field or remove zoneId, partId ones. From my point of view, the latter is better here.